game/graphics/opengl_renderer/Shader.cpp

```diff
@@ -64,6 +67,23 @@ Shader::Shader(const std::string& shader_name, GameVersion version) : m_name(sha
     return;
   }
 
+  // uniform samplers must be named matching the texture unit
+  glUseProgram(m_program);
+  for (int i = 1; i < 30; ++i) {
+    std::string uniformName = "tex_T" + std::to_string(i);
+    GLint texLoc = glGetUniformLocation(m_program, uniformName.c_str());
+    if (texLoc != -1) {
+      lg::debug("Shader {}: setting up texture uniform {}", shader_name, uniformName);
+    }
+    glUniform1i(texLoc, i);
```

Comment:
I think this `glUniform1i` should be inside the `if` statement above

Reply:
oh, yeah that was my intention as well! good catch